Source/WebCore/page/PointerLockController.cpp

```diff
@@ -40,44 +42,93 @@
 
 namespace WebCore {
 
+class UnadjustedMovementPlatformMouseEvent : public PlatformMouseEvent {
+public:
+    UnadjustedMovementPlatformMouseEvent(const PlatformMouseEvent& src)
+        : PlatformMouseEvent(src)
+    {
+        m_movementDelta = src.unadjustedMovementDelta();
+    }
+};
+
 PointerLockController::PointerLockController(Page& page)
     : m_page(page)
 {
 }
 
-void PointerLockController::requestPointerLock(Element* target)
+PointerLockController::~PointerLockController()
```

Comment:
`PointerLockController::~PointerLockController() = default;`

Reply:
This won't work. See the resolved discussion above: https://github.com/WebKit/WebKit/pull/17525#discussion_r1320203577

Reply:
Did you misunderstand the feedback and marked it as default in the header instead of the cpp? My suggestion is correct. If you need to include a header in the cpp to make it build, so be it, just include that header. That's not an issue in the cpp.

Reply:
Oh, I misunderstood your comment. I think this will work. Will fix in next push.

Reply:
To be able to forward declare in the header instead of adding an include, you just need to default the destructor in the cpp, you don't actually need a destructor body.